docs/en/compute-edition/32/admin-guide/compliance/custom-compliance-checks.adoc

```diff
@@ -6,17 +6,18 @@ Prisma Cloud lets you implement your custom image checks with simple scripts.
 
 Custom compliance checks are supported for:
 
-- Linux and Windows hosts (Host configured for docker, containerd, or CRI-O)
-- Docker images on Linux hosts
-- OCI images
+* Linux and Windows hosts (Host configured for docker, containerd, or CRI-O)
+* Docker images on Linux hosts
+* OCI images
 
 Custom compliance checks are not supported for:
 
-- Agentless scanning on Windows hosts
-- Linux and Windows containers
-- Docker images on Windows hosts
-- Tanzu Application Service (TAS) defender
-- GKE Autopilot
+* Agentless scanning on Windows hosts
+* Linux and Windows containers
+* Docker images on Windows hosts
+* Tanzu Application Service (TAS) defender
+* GKE Autopilot
+* Custom host compliance is not supported on Windows
```

Comment:
@anaghapamidi I have added a comment in Jira about changing this to "Windows hosts". Please change this to what Alexander suggests.

Reply:
The change is updated @manukumar6 - PTAL